omnibus/.kitchen.yml

```diff
@@ -9,26 +9,15 @@ driver:
 
 provisioner:
   name: chef_zero
-  require_chef_omnibus: 12.13.37
+  require_chef_omnibus: 13.6.4
 
 platforms:
-  - name: ubuntu-12.04
-    run_list: apt::default
+  - name: ubuntu-14.04
   - name: centos-5.11
-    run_list: yum-epel::default
   - name: centos-6.8
```

Comment:
You should make this centos-6 and centos-7 which always point to the latest point release.

Reply:
YES I SHOULD